frames/src/main/java/com/checkout/frames/component/billingaddressfields/BillingAddressInputComponentState.kt

```diff
@@ -0,0 +1,7 @@
+package com.checkout.frames.component.billingaddressfields
+
+import com.checkout.frames.component.base.InputComponentState
+
+internal data class BillingAddressInputComponentState(
+    val mappedInputComponentState: Map.Entry<String, InputComponentState>
```

Comment:
It's pretty unusual to use map Entry in such a way.
Could we just use 2 self-describable fields instead?

Reply:
Yes, it's a far better idea to do, reducing unnecessary public interfaces for the merchants. Excellent catch and learning for it.